app/controllers/recipes_controller.rb

```diff
@@ -54,7 +54,7 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_recipe
-      @recipe = current_user.recipes.find(params[:id])
```

Comment:
Since we have added the dashboard, which shows all the recipe and from there if i click on edit button on selected recipe, it throw me error if i put current_user.recipes